Database/SQLite/Simple/FromField.hs

```diff
@@ -1,6 +1,7 @@
-{-# LANGUAGE CPP, DeriveDataTypeable, DeriveFunctor  #-}
-{-# LANGUAGE FlexibleInstances, TypeSynonymInstances #-}
-{-# LANGUAGE ScopedTypeVariables      #-}
+
```

Comment:
Extra newline. Maybe you inherited it from the recently reverted commit.

Reply:
OK, removed.